sunpy/coordinates/ephemeris.py

```diff
@@ -109,15 +113,110 @@ def get_earth(time='now'):
     return earth
 
 
-def get_sun_B0(time='now'):
+@add_common_docstring(**_variables_for_parse_time_docstring())
+def get_horizons_coord(body, time='now', id_type='majorbody'):
+    """
+    Queries JPL HORIZONS and returns a `~astropy.coordinates.SkyCoord` for the location of a
+    solar-system body at a specified time.  This location is the instantaneous or "true" location,
+    and is not corrected for light travel time or observer motion.
+
+    .. note::
+        This function requires the Astroquery package to be installed and
+        requires an Internet connection.
+
+    Parameters
+    ----------
+    body : `str`
+        The solar-system body for which to calculate positions
+    id_type : `str`
+        If 'majorbody', search by name for planets or satellites.  If 'id', search by ID number.
```

Comment:
Same here - what does ID number mean and where can I find out what it means and what are acceptable values?